src/main/java/koreatech/in/domain/NotiSlack.java

```diff
@@ -51,9 +52,12 @@ public static NotiSlack deleteCompleteNotiSlack(User user) {
     }
 
     public static NotiSlack ownerShopRequestNotiSlack(User user) {
+        String newLine = System.lineSeparator();
+        String appendedTextAdminPageUrl = OWNERSHOP_REQUEST_SUFFIX + newLine + MARKDOWN_ADMIN_PAGE_URL;
+
         return NotiSlack.builder()
                 .color(NotiSlack.COLOR_GOOD)
-                .text(makeText(user, OWNERSHOP_REQUEST_SUFFIX))
+                .text(makeText(user, appendedTextAdminPageUrl))
```

Comment:
```suggestion
                .text(makeText(user, OWNERSHOP_REQUEST_SUFFIX + System.lineSeparator() + MARKDOWN_ADMIN_PAGE_URL))
```
는 어떨까요? (지역변수 제거하구)
여러 지역변수들이 별 책임을 가지진 않은 듯해서 여쭤봅니다.

일단 크게 상관은 없을 것 같아 어프로브했어요

Reply:
가독성 측면에서 makeText함수의 인자가 너무 길어지는 것을 염려했습니다. 문자열의 조합에 의미있는 이름을 부여하기 위해 지역변수를 사용했는데 단순히 이름부여를 위해 지역변수를 선언하는 것은 비효율적인가요??